# Disallows usage of Clarity Angular cards (no-clr-card)

The use of Clarity Angular cards is discouraged. Use Clarity Core cards instead.

## Rule Details

Examples of **incorrect** code for this rule:

```html
<div class="card">1</div>
```

Examples of **correct** code for this rule:

```html
<cds-card>1</cds-card>
```
